games-action/bugdom/additional-files/bugdom.rdef.in

```diff
@@ -0,0 +1,69 @@
+resource app_flags B_MULTIPLE_LAUNCH;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = 0,
```

Comment:
The template rdef.in file you started with doesn't seem to be using the MINOR part of the version.